notebooks/tachyon/Simple Test1.snb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
saved snb when local debug.  there are no problem when running.
add notebooks to .ignore. when need change notebooks' snb file, close the ignore first. OK?